website/content/docs/commands/secrets/disable.mdx

```diff
@@ -30,3 +30,20 @@ $ vault secrets disable aws/
 
 There are no flags beyond the [standard set of flags](/docs/commands)
 included on all commands.
+
+## Force Disable
+
+Because `secrets disable` revokes secrets associated with this mount, there can
+be errors that prevent the secrets engine from being disabled if the revocation
+fails.
+
+The best way to resolve this is to figure out the underlying issue, and then
+disable the secrets engine once that has been solved. Often, this can be as
```

Comment:
'that' is an antecedent, so does 'that' refer to the underlying issue here? If so, consider rewriting:

The best way to resolve this is to figure out the underlying issue, and then disable the secrets engine once the underlying issue is resolved.